src/main/webapp/app/entities/programming-exercise/programming-exercise.component.ts

```diff
@@ -15,15 +15,20 @@ import { AccountService } from 'app/core/auth/account.service';
 import { NgbModal } from '@ng-bootstrap/ng-bootstrap';
 import { ProgrammingExerciseImportComponent } from 'app/entities/programming-exercise/programming-exercise-import.component';
 import { FeatureToggle } from 'app/feature-toggle';
+import { IntelliJState, isIntelliJ } from 'app/intellij/intellij';
+import { JavaBridgeService } from 'app/intellij/java-bridge.service';
+import { stringifyCricular } from 'app/shared/util/utils';
 
 @Component({
     selector: 'jhi-programming-exercise',
     templateUrl: './programming-exercise.component.html',
 })
 export class ProgrammingExerciseComponent extends ExerciseComponent implements OnInit, OnDestroy {
-    FeatureToggle = FeatureToggle;
     @Input() programmingExercises: ProgrammingExercise[];
     readonly ActionType = ActionType;
+    readonly isIDE = isIntelliJ;
```

Comment:
please use consistently isIntelliJ or isIDE

Reply:
Renamed to `isIntelliJ`